src/sentry/services/hybrid_cloud/organization/impl.py

```diff
@@ -235,3 +251,41 @@ def _query_organizations(
             return [r.organization for r in results if scope in r.get_scopes()]
 
         return [r.organization for r in results]
+
+    @staticmethod
+    def _deserialize_member_flags(flags: ApiOrganizationMemberFlags) -> int:
+        return flags_to_bits(flags.sso__linked, flags.sso__invalid, flags.member_limit__restricted)
+
+    def add_organization_member(
+        self,
+        *,
+        organization: ApiOrganization,
+        user: APIUser,
+        flags: ApiOrganizationMemberFlags | None,
+        role: str | None,
+    ) -> ApiOrganizationMember:
+        member = OrganizationMember.objects.create(
+            organization_id=organization.id,
+            user_id=user.id,
+            flags=self._deserialize_member_flags(flags) if flags else 0,
+            role=role or organization.default_role,
+        )
+        return self.serialize_member(member)
+
+    def add_team_member(
+        self, *, team_id: int, organization_member: ApiOrganizationMember
+    ) -> ApiTeamMember:
+        omt = OrganizationMemberTeam.objects.create(
+            team_id=team_id, organizationmember_id=organization_member.id
+        )
+        project_ids = ()  # TODO?
```

Comment:
Should project_ids be a parameter?

Reply:
Per discussion with @corps, we'll just sidestep this by not having a return value: 3c3e5538d8761cd6ab559e74ba0a9eb4dac9cd13